example/src/main/java/com/stripe/example/Settings.kt

```diff
@@ -62,6 +62,13 @@ class Settings(context: Context) {
          */
         private val STRIPE_ACCOUNT_ID: String? = null
 
+        // Example payment sheet backend with a "/checkout" endpoint
+        // Remix from https://glitch.com/edit/#!/stripe-mobile-payment-sheet-test-playground-v3
+        internal const val PAYMENT_SHEET_BASE_URL = ""
```

Comment:
should we put in `"https://example.com"` or some other placeholder value so make it clear what should go in here?